packages/pf4-component-mapper/src/files/dual-list-select.d.ts

```diff
@@ -25,15 +35,81 @@ interface InternalDualListSelectProps {
   allToRight?: boolean;
   moveAllLeftTitle?: ReactNode;
   moveAllRightTitle?: ReactNode;
+  label?: ReactNode;
+  isRequired?: boolean;
+  helperText?: ReactNode;
   noValueTitle?: ReactNode;
   noOptionsTitle?: ReactNode;
   filterOptionsTitle?: ReactNode;
   filterValueTitle?: ReactNode;
   filterValueText?: ReactNode;
   filterOptionsText?: ReactNode;
-  leftValues: DualListSelectOption[];
-  rightValues: DualListSelectOption[];
+  description?: ReactNode;
+  hideLabel?: boolean;
+  id?: string;
+  state?: AnyObject,
+  sortOptions?: VoidFunction,
+  filterOptions?: VoidFunction,
+  leftValues?: DualListSelectOption[];
+  rightValues?: DualListSelectOption[];
   renderStatus?: RenderStatusFunction;
+  handleOptionsClick?: HandleOptionsFunction,
```

Comment:
all of these handlers, sorter, etc, should not be part of the interface. These functions are inject from the common dual list select implementation imported from
```jsx
import DualListSelectCommon from '../../../common/src/dual-list-select';
```
and used at
```jsx
const DualListSelectWrapper = (props) => <DualListSelectCommon {...props} DualListSelect={DualList} />;
```
You can't rewrite those from the schema. Only props configurable from schema should be listed here.

I think any the stuff after the `// props` comment in the prop types definition.